pixart.js

```diff
@@ -1 +1,34 @@
+var button = $("#set-color");
 
+button.on("click", colorCheck);
+button.on("click", preventReset);
+
+//*checks for color
+function colorCheck() {
+// alert($("input").val());
+$(".brush").css("background", $("input").val());
+
+}
+
+function preventReset() {
+  event.preventDefault();
+
+}
+.
+var pageLimit=20;
+
+$(document).ready(function () {
+         for (var i = 0; i < pageLimit; i++) {
+             $('body').append('<div class= "square"></div>');
+         }
+     });
+
+
+$(".square").on("click", function () {
+$(this).css("background", "green");
+})
+
```

Comment:
move these guys Lines 27 & 28 up. but i think we did that in person the other day and this isn't your most recent code :)